src/js/constants.ts

```diff
@@ -0,0 +1,242 @@
+export const MediaUIEvents = {
```

Comment:
tried to keep these as tight as possible to make derivation of types automatic and easier to maintain as we add things.